app/api/users_events_roles.py

```diff
@@ -37,10 +39,27 @@ class UsersEventsRolesDetail(ResourceDetail):
     users_events_roles detail by id
     """
 
-    decorators = (api.has_permission('is_coorganizer', methods="GET,PATCH,DELETE"),)
+    def before_delete_object(self, users_events_roles, view_kwargs):
+        """
+        method to check for proper permissions for deleting
+        :param users_events_roles:
+        :param view_kwargs:
+        :return:
+        """
+        if not current_user.is_staff and users_events_roles.role_id == 7:
```

Comment:
What is role ID 7?

Reply:
Since initial roles aren't allowed to be deleted so I set 7 which denotes role owner in the roles table.

Reply:
In my local machine, it is 5, on production, it is 12!
What guarantee is there that owner role ID is going to remain 7 on all machines, DB, forever?

Reply:
ohh didn't think of that :sweat_smile: will update :v: